### PR TITLE
Fix opencv and tesserocr

### LINKED ISSUE #364: selection via OCRD_MODULES fails due to dependent modules
I tried to install ocrd_all with the setup-guide: https://ocr-d.de/en/setup:

    make all OCRD_MODULES="core ocrd_tesserocr ocrd_cis"

and got following error running the mentioned command:
```
Successfully installed numpy-1.24.3 shapely-2.0.1
make[1]: Leaving directory '/home/cloud/repos/ocrd_all/core' 
. /home/cloud/repos/ocrd_all/venv/bin/activate && cd tesserocr && sem -q --will-cite --fg --id ocrd_all_pipvenv pip insta
ll  .
ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
make: *** [Makefile:671: /home/cloud/repos/ocrd_all/venv/share/tesserocr] Error 1
```
Seems the problem is that ocrd_tesserocr needs tesserocr to be installed. The installation of tesserocr is triggered but the sources are not fetched and that leads to the error. Running `make all OCRD_MODULES="core ocrd_tesserocr ocrd_cis tesseract tesserocr"` resolves the problem but maybe it would be good to change the  makefile to be able to fetch the needed sources itself.

### PR DESCRIPTION
**Pull request by bertsky**
fixes #364

### CODE REVIEW

**Comment by bertsky** [comment]
@kba ping

File: Makefile [hunk 1]
```diff
@@ -184,26 +184,27 @@ wheel: $(BIN)/wheel
...
 .PHONY: ocrd
 ocrd: $(BIN)/ocrd
+ifneq ($(filter core, $(OCRD_MODULES)),)
 deps-ubuntu-modules: core
 $(BIN)/ocrd: core
 	. $(ACTIVATE_VENV) && $(MAKE) -C $< install PIP="$(SEMPIP) pip" PIP_INSTALL="$(SEMPIP) pip install $(PIP_OPTIONS)" && touch -c $@
+else
+CUSTOM_DEPS += python3 imagemagick libgeos-dev
+$(BIN)/ocrd: | $(ACTIVATE_VENV)
+	. $(ACTIVATE_VENV) && $(SEMPIP) pip install $(PIP_OPTIONS_E) ocrd ocrd_network
```
Comment (kba) [inline Makefile:198]: `ocrd` does imply `ocrd_network`, so is redundant.

**Review by kba — APPROVED** [review]
LGTM, funny for how long nobody noticed the `fundstring`/`filter` differences too.

File: Makefile (hunk 1 above)
Comment (bertsky) [reply Makefile]: https://github.com/bertsky/ocrd_all/commit/90fc08bcea4774e9b4caa95a7c241bdbee026fb2

**Comment by bertsky** [comment]
Has been merged into #362

**Pull request closed without merging**